src/app/pages/journal-column/journal-column.component.scss

```diff
@@ -0,0 +1,72 @@
+@import "src/vars";
+
+:host {
+  display: grid;
+  grid-gap: 1rem;
+  padding-right: 1rem;
+}
+.back-link {
+  position: relative;
+  left: 0.5rem;
+  text-transform: uppercase;
+  font-weight: 700;
+  font-size: .875rem;
+  color: #777777;
+  transition: $transition;
+
+  .arrow-back {
+    position: absolute;
+    top: -0.05rem;
+    left: -0.5rem;
+    color: inherit;
+  }
+
+  &:hover {
+    color: $accent;
+  }
+}
+
+.topic-value {
+  width: 88%;
+  padding: 0rem 1rem 2rem 1rem ;
+}
+
+.header {
+  display: grid;
+  grid-template-columns: calc(3rem + 2px) 1fr 8rem 8rem 8rem 0.25fr;
+  align-content: center;
+  grid-column-gap: 1rem;
+
+  .indexs {
+    display: grid;
+    align-content: center;
+    font-weight: bold;
+    width: calc(3rem + 2px);
+    height: calc(3rem + 2px);
+    background-color: $accent;
+    color: #ffffff;
+    text-align: center;
+  }
+  .full-name {
+    align-self: center;
+    text-align: left;
+    line-height: 2rem;
+    font-weight: bold;
+    background-color: $light;
+    border: 1px solid $accent;
+    padding: 0.5rem; 
+  }
+  .select-mark {
+    align-self: center;
+    text-align: center;
+    line-height: 2rem;
+    font-weight: bold;
+    background-color: $light;
+    border: 1px solid $accent;
+    padding: 0.5rem; 
+  }
+}
+.finish {
+  position: relative;
+  left: 89%;
+}
```

Comment:
The whole file is something weird, unstructured, copied and pasted. Also I think there are some useless styles.